test/fixtures/rules/std_rule_cases/L052.yml

```diff
@@ -77,18 +117,37 @@ test_pass_multi_statement_no_trailing_semi_colon_custom_require:
       L052:
         require_final_semicolon: True
 
-test_fail_multi_statement_no_trailing_semi_colon_custom_require_newline:
+test_pass_multi_statement_no_trailing_semi_colon_custom_require_oneline:
```

Comment:
What's happening here? The test name says "pass", but it provides a "fail_str" and a "fix_str"? FWIW, the behavior looks correct; only the test _name_ seems wrong.

Reply:
ah sorry about that, easy to one of these when making all the various permutations 😄